Sources/Utils/Locate.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+#if swift(>=5.1)
+@propertyWrapper
+public struct Locate<Service> {
+    private var service: Service
+    public init(name: String? = nil, locator: ServiceLocator = .shared) {
+        do {
+            self.service = try locator.get(name: name)
+        } catch {
+            fatalError(error.localizedDescription)
+        }
+    }
+    public var wrappedValue: Service {
+        get { return service }
+        mutating set { service = newValue }
+    }
+    public var projectedValue: Locate<Service> {
```

Comment:
I might be wrong 🙏, but I believe that considering that we are projecting _self_ and we do not have any public parameter or method, we could remove _projectedValue_ from here 🤔.
For more information: https://docs.swift.org/swift-book/LanguageGuide/Properties.html